### ISSUE TITLE
Parameter change cuts audio

### ISSUE THREAD

**Issue by spacelordmother** [coffeeshopped/casio-cz-editor#1]
Got the app open and running - tested connection (using 2 different interfaces and also with Ableton to double check midi message pass-through) with my CZ-101 and all seems good there.  App doesn't read the current settings from the synth on startup or with button press, and then trying to adjust any parameter kills the sound.  At that point I can change the patch on the CZ and operation resumes as normal until I try to change another parameter in the app.  

Strangely, the only setting that works is to scroll the tuning value in the app -- the patch doesn't crash and I can hear the pitch raise or lower as it should.

**Comment by coffeeshopped** [comment]
Hm, the app works on my end. There could be an issue with the sysex data getting garbled by being sent by more than one app at a time... especially if you have Ableton running. Ableton won't pass on sysex data correctly.

**Comment by spacelordmother** [comment]
I had Ableton closed at the time - I only used it to make sure that I could pass midi the the CZ on the in and outs of my interfaces

**Comment by spacelordmother** [comment]
To follow up -- I downloaded the editor on my laptop and it worked great.  Then I tried again on my desktop and still had the same issue as before.  I noticed in the max window that it couldn't find a presets file so I deleted everything and tried fresh.  Now it works!  

Thanks so much for this -- I love the CZ sound, but editing is/was so tedious that I was thinking of passing it along.  Now she can stay!
